### PR TITLE
Update `ja` localized messages

### PR DESCRIPTION
**Pull request by Gudahtt**
These translations were provided by Lionbridge.

### CODE REVIEW

**Comment by Gudahtt** [comment]
If any community contributors would like to help review this PR, it would be greatly appreciated!

I'll tag anyone here that has contributed in the past: @nyatla @mamiM 

I wanted to give you all the opportunity to provide feedback because we owe our current translations to you. Please don't feel obliged if you don't have the time to review, and please let me know if you'd like me to stop tagging you for reviews in the future.

Thanks!

**Comment by nyatla** [comment]
I'll review it now and make a fix list.

**Comment by nyatla** [comment]
I started checking and fixing.
Progress is 116/687.
You can find it in this repository.
https://github.com/nyatla/metamask_PR10265/tree/master/message_translation_review20200126
Give me a little more time.

**Comment by Gudahtt** [comment]
Thank you @nyatla - take as much time as you need!

**Comment by nyatla** [comment]
memo.
Please let me know if anyone have any comments.

1. I feel that the description of connectToAllAccounts is incorrect.
2.I'm trying to make "contacts" into an address book. (I will check the source later.)

**Comment by Gudahtt** [comment]
>2.I'm trying to make "contacts" into an address book.

Ah, great catch! That sounds good.

>I feel that the description of connectToAllAccounts is incorrect.

Could you elaborate? Do you mean this description:

>"will replace $1 in connectToAll, completing the sentence 'connect to all of your accounts', will be text that shows list of accounts on hover"

That is correct as far as I know :thinking:. It's not a well written description though. I would update it to say:

>"This will replace $1 in connectToAll, completing the sentence 'connect to all of your accounts'. This text will show a list of accounts on hover."

The hover behaviour is why that piece is translated separately; because we want to make a tooltip appear with additional information when the user hovers the mouse over that piece of text.

**Comment by Gudahtt** [comment]
As an aside, I noticed that you wrote an interesting looking script to help with your efforts here. If there's any way we can make this process easier, I'd be interested in building a script like that into this repo directly.

We already have some scripts meant for working with these files. `development/verify-locale-strings.js` will validate and normalize these files, and `development/missing-locale-strings.js` will highlight any missing messages. But we don't have any script yet that puts the english translation alongside each message for comparison. That sounds potentially useful :thinking:

**Comment by nyatla** [comment]
I am sorry.
It is not description.

I thought it was a mistake because the current en message of "connectToAllAccounts" was "accounts".

**Comment by nyatla** [comment]
backupApprovalInfo
backupApprovalNotice

Those are Secret Code / Secret, but this probably looks like a seed phrase.

**Comment by nyatla** [comment]
1.rpcUrl is the New RPC URL. But it seems to the RPC URL.
2.endOfFlowMessage6
now: If you need to back up your seed phrase again, you can find it in Settings -> Security.
update: If you need to back up your seed phrase again, you can find it in Settings -> Security & Privacy.

**Comment by nyatla** [comment]
Currently , 601/687 translated.
I will finished it in a few days.

**Comment by Gudahtt** [comment]
This PR has been rebased onto `develop` to ensure that the `yourSigRequested` message was not present (it was removed in #10300).

@nyatla Sure, a diff would be great! I can prepare the changes for GitHub myself.

The rebase I just did should hopefully not affect your work, since only one message was removed, and it was at the very end of the file.

File: app/_locales/ja/messages.json [hunk 1]
```diff
@@ -1,478 +1,2102 @@
...
+  },
+  "fastest": {
+    "message": "最も高速"
+  },
+  "feeAssociatedRequest": {
+    "message": "手数料はこのリクエストに関連付けられています。"
   },
   "fiat": {
-    "message": "法定通貨",
+    "message": "フィアット",
```
Comment (kumavis) [inline app/_locales/ja/messages.json:667]: 一般人わかるかな :sweat_smile:

**Review by kumavis — CHANGES_REQUESTED** [review]
せいわ なっております :bow:

File: app/_locales/ja/messages.json (hunk 1 above)
Comment (nyatla) [reply app/_locales/ja/messages.json:667]: For Japanese localization, I am working on my repository.
I will check the item that was pointed out.

https://github.com/nyatla/metamask_PR10265/blob/master/message_translation_review20200126/message.json

File: app/_locales/ja/messages.json [hunk 2]
```diff
@@ -1,478 +1,2102 @@
...
+  "decryptInlineError": {
+    "message": "このメッセージは次のエラーによって復号できません。$1",
+    "description": "$1 is error message"
+  },
+  "decryptMessageNotice": {
+    "message": "$1 は、このメッセージを読んでアクションを完了することを望んでいます",
+    "description": "$1 is the web3 site name"
+  },
+  "decryptMetamask": {
+    "message": "メッセージの復号"
```
[the remark this answers is not in the document]
Comment (Gudahtt) [reply app/_locales/ja/messages.json:448]: Should be `decrypedMessage`; this was a poor choice in message name.

File: app/_locales/ja/messages.json [hunk 3]
```diff
@@ -1,478 +1,2102 @@
...
+  },
+  "decryptRequest": {
+    "message": "リクエストの復号"
```
[the remark this answers is not in the document]
Comment (Gudahtt) [reply app/_locales/ja/messages.json:451]: It's the latter in this case. We should definitely add descriptions for those though, agreed.

**Comment by nyatla** [comment]
100% translated.

File: app/_locales/ja/messages.json [hunk 4]
```diff
@@ -1,478 +1,2102 @@
...
-    "message": "署名しています。"
+    "message": "著名しています"
+  },
+  "yourPrivateSeedPhrase": {
+    "message": "秘密のシード フレーズ"
+  },
+  "zeroGasPriceOnSpeedUpError": {
+    "message": "スピードアップ時にゼロのガス価格"
   }
 }
```
Comment (nyatla) [inline app/_locales/ja/messages.json:2102]: Thank you for waiting.
This is 100% coverage file.It is updated for #10300.
please  replace messages.js file.

https://github.com/nyatla/metamask_PR10265/blob/master/message_translation_review20200126/messages.json

**Review by nyatla — CHANGES_REQUESTED** [review]

File: app/_locales/ja/messages.json [hunk 5]
```diff
@@ -1,478 +1,2102 @@
...
+    "message": "高度なガス コントロール"
+  },
+  "showAdvancedGasInlineDescription": {
+    "message": "これを選択すると、ガス価格とリミット コントロールが送金画面と確認画面に直接表示されます。"
+  },
+  "showFiatConversionInTestnets": {
+    "message": "テストネット上に変換を表示"
+  },
+  "showFiatConversionInTestnetsDescription": {
+    "message": "これを選択すると、テストネット上にフィアット変換が表示されます"
```
Comment (kumavis) [inline app/_locales/ja/messages.json:1462]: "fiat" used here

File: app/_locales/ja/messages.json (hunk 4 above)
Comment (kumavis) [reply app/_locales/ja/messages.json:2102]: @Gudahtt @nyatla's version has more differences than appear in the above suggested change. we should dump that message.json on top of this and review the diff again

File: app/_locales/ja/messages.json (hunk 5 above)
Comment (nyatla) [reply app/_locales/ja/messages.json:1462]: @kumavis
フィアット looks like a word that has not yet been recognized in Japan. So the 法定通貨 looks good.
However,  At the  place, Fiat may be a better expression.

@Gudahtt @kumavis
My file has a lot of changes. I'll make a new pull request and explain the changes.

**Comment by nyatla** [comment]
This is pull request.
https://github.com/MetaMask/metamask-extension/pull/10362

**Comment by kumavis** [comment]
blocked: need CLA approval from nyatla

**Comment by kumavis** [comment]
rebased for storybook CI :grin:

**Comment by kumavis** [comment]
some strange errors, should be unrelated
```
[2021-02-06 06:04:30.873][e2e]       AssertionError [ERR_ASSERTION]: '21.00025' == '0.00025'
```

**Comment by nyatla** [comment]
Is this error bt my patch?
Would you give me a hint of the error?

**Comment by kumavis** [comment]
@nyatla no the problem was not related to your changes

**Comment by kumavis** [comment]
you can see part of the ja localization here
need to select `ja` from the localization dropdown at the top toolbar (globe icon)
https://242415-42009758-gh.circle-artifacts.com/0/storybook/index.html?path=/story/feecard--with-all-props

**Comment by nyatla** [comment]
@kumavis OK, Thank you.

File: app/_locales/ja/messages.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (kumavis) [reply app/_locales/ja/messages.json]: @nyatla `復号のリクエスト` sound ok here?

File: app/_locales/ja/messages.json [hunk 6]
```diff
@@ -62,103 +82,400 @@
...
+  },
+  "decryptMessageNotice": {
+    "message": "$1 はメッセージを読み取ってアクションを完了しようとしています",
+    "description": "$1 is the web3 site name"
+  },
+  "decryptMetamask": {
+    "message": "メッセージを復号"
+  },
+  "decryptRequest": {
+    "message": "リクエストを復号"
```
Comment (kumavis) [inline app/_locales/ja/messages.json:460]:
```suggestion
    "message": "復号のリクエスト
```

Comment (kumavis) [reply app/_locales/ja/messages.json:460]: this should be "request for decryption"
@nyatla `復号のリクエスト` sound ok here?

**Comment by nyatla** [comment]
@kumavis

This looks like the header text.

There are two interpretations.

1. When the user requests Metamask to decrypt "something"
復号をリクエスト

2. When the user requests Metamask to decrypt the any "request"
リクエストを復号

I could not run how this component works, so I selected 2.

**Review by kumavis — DISMISSED** [review]
👌

**Pull request merged**